### ISSUE TITLE
TextInput with suggestions - voiceover navigation

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7328]
### Expected Behavior
It should be easy and clear for a voiceover screanreader user to navigate to the suggestions

### Actual Behavior
Navigating to the TextInput suggestions while using voiceover (I havn't tested with other browsers) is not a good experience. It is not clear to the screenreader user what keys need to be pressed to navigate to suggestions or how to get back to the input.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/input-textinput-suggestions--suggestions

### Steps to Reproduce
Test the above story with voice over

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by jcfilben** [comment]
Resolved in https://github.com/grommet/grommet/issues/7549
